CHANGELOG.md

```diff
@@ -46,6 +46,7 @@
 * [#4578](https://github.com/bbatsov/rubocop/issues/4578): Fix false positive in `Lint/FormatParameterMismatch` for format with "asterisk" (`*`) width and precision. ([@smakagon][])
 * [#4285](https://github.com/bbatsov/rubocop/issues/4285): Make `Lint/DefEndAlignment` aware of multiple modifiers. ([@drenmi][])
 * [#4634](https://github.com/bbatsov/rubocop/issues/4634): Handle heredoc that contains empty lines only in `Layout/IndentHeredoc` cop. ([@pocke][])
+* [#4593](https://github.com/bbatsov/rubocop/issues/4593): Fix false positive in "Rails/SaveBang" when 'save/update_attribute' is used with a 'case' statement. ([@theRealNG][])
```

Comment:
The changelog entry doesn't match what the PR actually does.

The name of the cop and code references should have been in backticks, not quotes.